x11-misc/emwm-utils/emwm-utils-9999.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Utilities for Enhanced Motif Window Manager (EMWM)."
+HOMEPAGE="http://fastestcode.org/emwm.html"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/alx210/mwm-utils.git"
+else
+	SRC_URI="https://fastestcode.org/dl/emwm-utils-src-1.0.tar.xz -> ${P}.tar.xz"
```

Comment:
1.0?

Also, while renaming isn't _wrong_ per se here, it has two issues:
1. _not_ renaming would allow pkgcheck to warn as it notices hardcoded version numbers
2. it makes it harder to look up distfiles in the distant future by googling, imagine if gone from mirrors etc but someone really wants this copy